src/pyodide/python-entrypoint-helper.ts

```diff
@@ -162,26 +163,98 @@ function makeHandler(pyHandlerName: string): Handler {
     }
   };
 }
+
+function makeDurableObjectClass(className: string, classKind: AnyClass) {
+  class DurableObjectWrapper extends classKind {
+    pyInstance: Promise<PyModule>;
+
+    constructor(...args: any[]) {
+      super(...args);
+      // Initialise a Python instance of the class.
+      this.pyInstance = this.initPyInstance(args);
+    }
+
+    async initPyInstance(args: any[]) {
+      const mainModule = await getMainModule();
+      const constructor = mainModule[className] as unknown as (
```

Comment:
Can we call this `pyClass` maybe? It's confusing with the `constructor()` function above.